roles/hammerdb/tasks/main.yaml

```diff
@@ -21,8 +21,7 @@
         name: '{{ meta.name }}-workload-{{ trunc_uuid }}'
         namespace: '{{ operator_namespace }}'
       data:
-        workload.tcl: "{{ lookup('template', 'workload.tcl.j2') }}"
-
+ "{{ lookup('template', 'tpcc-workload.tcl.j2') }}"
```

Comment:
using lookup in this manner will make the logs difficult to readable as it'll literally copy paste the template here, can we instead template the creation of configmap so that the task looks neater and cleaner? just a suggestion, but it'll help greatly while debugging.